app/src/main/res/menu/menu_navigation.xml

```diff
@@ -2,37 +2,25 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <group android:id="@+id/menu_findGame_group">
-        <item
-            android:id="@+id/menu_actionSearch"
-            android:orderInCategory="200"
-            android:title="@string/searchFilterSort"
-            android:icon="@drawable/magnifyingglass"
-            app:showAsAction="ifRoom"/>
-    </group>
-
     <group android:id="@+id/menu_gm_group">
         <item
             android:id="@+id/menu_createGame"
             android:icon="@drawable/ic_quill"
             android:orderInCategory="200"
-            android:title="@string/title_create_game_activity"
-            app:showAsAction="ifRoom" />
+            android:title="@string/title_create_game_activity"/>
```

Comment:
Format using camel case like the others

Reply:
I thought it would be better to use snake case in the xml files since it's the xml convention